src/Resources/config/grids/setono_sylius_analytics_admin_property.yaml

```diff
@@ -9,6 +9,11 @@ sylius_grid:
                 trackingId:
                     type: string
                     label: setono_sylius_analytics.ui.tracking_id
+                enabled:
+                    type: twig
+                    label: setono_sylius_analytics.ui.enabled
```

Comment:
Maybe there's a Sylius translation for this? `sylius.ui.enabled` maybe

Reply:
Yes, it's true, I did it because I saw a "local" translation in your bundle.

Don't worry, I can change the label key.